package.json

```diff
@@ -1,183 +1,134 @@
 {
-  "name": "terra-application",
-  "main": "lib/application-base/ApplicationBase.js",
-  "version": "1.48.0",
-  "description": "A framework to support application development with Terra components",
+  "name": "terra-application-mono",
```

Comment:
Should we keep the name as `terra-application` or are we actually renaming it to terra-application-mono?

Reply:
I needed to rename the base mono-repo's package to avoid some of terra-dev-sites aliasing. This shouldn't matter and we could change it at any time since the mono repo package itself won't ever get released.

terra-application will stay terra-application in it's package (that file shows up as new)

Reply:
ok. I now see [this](https://github.com/cerner/terra-application/pull/245/files#diff-e6b86f2cdec5e416120e26ad9582767f87da90eaad6f8cb232359c9e9cd26d48R2).